tests/LineBot.Tests/TestHelpers/TestHttpMessageHandler.cs

```diff
@@ -47,6 +47,12 @@ public TestHttpMessageHandler(HttpStatusCode statusCode)
             };
         }
 
+        public TestHttpMessageHandler(string stringData, bool isString)
```

Comment:
Why do you need the boolean when you don't use it?

Reply:
I want make a seperate MessageHandler, but don't know there is a better name..
Humm. will change this to a new method.

Reply:
```        public TestHttpMessageHandler(string responseFile)
            : this(HttpStatusCode.OK)
        {
            _response.Content = new StringContent(File.ReadAllText(responseFile));
        }```

To make sure not call this method..
Should we make another sub-class to implement it?

Reply:
You could also get a byte array from the response string instead?

Reply:
Oh. Cool, I didn't know that can be used in this way.
Changed.